bridge/hardhat/script/addFederatorMemeber.js

```diff
@@ -0,0 +1,41 @@
+// How to run the script: npx hardhat run ./hardhat/script/addFederatorMemeber.js --network bsctestnet,rsktestnetbsc
+const hre = require("hardhat");
+
+async function main() {
+  const {getNamedAccounts, deployments} = hre;
+  const {deployer} = await getNamedAccounts();
+  const transactionEtherValue = 0;
+  const memberFederatorAddress = "0xeac27e59f8a71613137e9c5d475d05c7d4d198e8";
+
+  const Federation = await deployments.get('FederationV2');
+  const FederationProxy = await deployments.get('FederationProxy');
```

Comment:
I know this is not a deployment, but can't we use the stuff in named accounts either way? we have access to a hre instance

Reply:
We can use the named accounts to get the addresses, but we would have to make changes in the functions that get the contract addresses because now we have 2 chunks of contract addresses for the same chain like the normal rsk testnet  and the rsk testnet for the binance contracts

Reply:
oh I see, maybe I'd just add an issue / Jira ticket to do that stuff in the future, thanks